### PR TITLE
add OwnerReference in kube_pod_info

### PR DESCRIPTION
**Pull request by zouyee**
fix [#139](https://github.com/kubernetes/kube-state-metrics/issues/139)

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kube-state-metrics/142)

### CODE REVIEW

File: collectors/pod.go [hunk 1]
```diff
@@ -39,7 +40,7 @@ var (
 	descPodInfo = prometheus.NewDesc(
 		"kube_pod_info",
 		"Information about pod.",
-		[]string{"namespace", "pod", "host_ip", "pod_ip", "node", "created_by"}, nil,
+		[]string{"namespace", "pod", "host_ip", "pod_ip", "node", "created_by", "owner_kind", "owner_name", "controller"}, nil,
```
Comment (brancz) [inline collectors/pod.go]: As a user I would expect the controller value to be something like "StatefulSetController", but it is actually a boolean value. I'd suggest, to rename it from "controller" to "owner_is_controller".

Comment (zouyee) [inline collectors/pod.go]: 👌

File: collectors/pod.go [hunk 2]
```diff
@@ -238,8 +239,16 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 	addCounter := func(desc *prometheus.Desc, v float64, lv ...string) {
 		addConstMetric(desc, prometheus.CounterValue, v, lv...)
 	}
+	owners := p.GetOwnerReferences()
+	if len(owners) == 0 {
+		addGauge(descPodInfo, 1, p.Status.HostIP, p.Status.PodIP, nodeName, extractCreatedBy(p.Annotations), "<none>", "<none>", "<none>")
+
```
Comment (zouyee) [inline collectors/pod.go]: nice check 😀

**Comment by andyxning** [comment]
@zouyee Could you please squash commits.

**Comment by zouyee** [comment]
thanks!

**Comment by brancz** [comment]
lgtm

**Pull request merged**